tasks/winchester.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Copy winchester schema to host
+  template: dest={{winchester_schema_file}} owner=root group=root mode=640 src=winchester.sql.j2
+  register: copy_result
+
+- name: Apply winchester schema to mysql db
+  shell: mysql -u{{mysql_admin}} -p{{mysql_admin_password}} < {{winchester_schema_file}} || echo "Did the schema change? This process will fail on schema changes."
```

Comment:
See the change from https://github.com/hpcloud-mon/ansible-monasca-schema/pull/12/files I think that should be done here also. The CS team is running their mysql with passwordless root access and it will choke if you specify -p